packs/bit-bucket/actions/list_user_repo.py

```diff
@@ -0,0 +1,21 @@
+from lib.action import BitBucketAction
+
+
+class GetRepos(BitBucketAction):
```

Comment:
This action returns a list so please call it `ListRepos` and `list_repos` (same as you do with `list_branches`).